cmd/retry-storage.go

```diff
@@ -260,15 +273,11 @@ func (f *retryStorage) reInitUponDiskNotFound(err error) bool {
 	return false
 }
 
-// Connect and attempt to load the format from a disconnected node,
-// attempts three times before giving up.
+// Connect and attempt to load the format from a disconnected node.
+// attempts maxRetryAttempts times before giving up.
```

Comment:
s/maxRetryAttempts/maxRetryAttempts + 1/ since `newRetryTimer` starts from zero.

Reply:
yes but we stop before `i < maxRetryAttempts`